testhelpers/steps.go

```diff
@@ -406,13 +406,25 @@ func (ctx *TestContext) TableAtIDShouldBe(tableName string, id int64, data *gher
 		if err := sqlRows.Scan(rowValPtr...); err != nil {
 			return err
 		}
+
+		nullValue := "null"
+		pNullValue := &nullValue
 		// checking that all columns of the test data table match the SQL row
 		for iCol, dataCell := range data.Rows[iDataRow].Cells {
 			colName := dataCols[iCol].Value
 			dataValue := dataCell.Value
-			sqlValue := rowValPtr[iCol].(*string)
-			if dataValue != *sqlValue {
-				return fmt.Errorf("not matching expected value at row %d, col %s, expected '%s', got: '%v'", iDataRow-1, colName, dataValue, *sqlValue)
+			sqlValue := rowValPtr[iCol].(**string)
+
+			if *sqlValue == nil {
+				sqlValue = &pNullValue
+			}
+
+			if dataValue == "true" && **sqlValue == "1" || dataValue == "false" && **sqlValue == "0" {
```

Comment:
I would prefer having parenthesis for this kind of expressions

Reply:
done